uiowa/uiowa
# SPPA -  News Story Migration

Resolves https://github.com/uiowa/uiowa/issues/3343

## To Test

- Sync D7 sppa (urban) site within grad application using Dev Desktop.
- Run the following and configure/save SiteNow Migrate
```
blt sync --site sppa.uiowa.edu && 
drush @sppa.local en sppa_migrate --yes && 
drush @sppa.local uli /admin/config/system/sitenow-migrate
```
```
database = urban
user = root
host = 10.0.2.2
port = 33067
base url = https://sppa.uiowa.edu
```
- `drush @sppa.local cr && drush @sppa.local migrate:status`
- You should see articles and redirects migrations
- `drush @sppa.local migrate:import --all`
- Watch as I ride the hard work of migration experts before me. The migration should be successful.
- Refresh the content overview. There are a lot of articles that got imported but only like three that are published. The first three published items all have images to check. The first image was imported.
- You should be redirected if you go to https://sppa.local.drupal.uiowa.edu/news/2021-03-01/new-certificate-in-urban-studies-for-undergraduates
